### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by sambukowski**
updates to the readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,34 +2,62 @@
...
+
+### Install dependencies
 
-# install dependencies
+```bash
 npm install -g dotenv-cli
 curl -L https://foundry.paradigm.xyz | bash
+```
+
+## Deployment
```
Comment (steezeburger) [inline README.md:26]: Not all the things in this section are Deployments. Some of them are scripts or tests. I'm not sure we need this.

File: README.md [hunk 2]
```diff
@@ -2,34 +2,62 @@
 
 This repository contains smart contracts and front end interfaces for the Astria Shared Sequencer.
 
-### Rough-draft how to
+## Installation
```
Comment (steezeburger) [inline README.md:5]: I don't think this section header is necessary.

**Review by steezeburger — CHANGES_REQUESTED** [review]
I don't think we need the `##` section headers.

File: README.md (hunk 2 above)
Comment (sambukowski) [reply README.md]: removed this

File: README.md (hunk 1 above)
Comment (sambukowski) [reply README.md]: removed this

File: README.md [hunk 3]
```diff
@@ -46,15 +42,17 @@ Deployed to: 0xA53639fB5458e65E4fA917FF951C390292C24A15
 Transaction hash: 0x775c63649c25d7b8029e2e786e001fcda1618723a305c2e384d5cae453a32ad7
 ```
 
-Copy the `Deployed to:` address to use for the next step.
+Copy the `Deployed to:` address to use for deploying Uniswap.
```
Comment (steezeburger) [inline README.md:45]: There should be a line between these two lines

**Review by steezeburger — APPROVED** [review]
Tiny formatting nit for README, but looks good besides that

**Pull request merged**